setup.cfg

```diff
@@ -35,7 +35,7 @@ install_requires =
     appdirs >= 1.4; platform_system!='Emscripten'
     bt-test-interfaces >= 0.0.2; platform_system!='Emscripten'
     click == 8.1.3; platform_system!='Emscripten'
-    cryptography == 39; platform_system!='Emscripten'
```

Comment:
I don't think that's going to work. As far as I can tell, Pyiodide only supported pure-python packages when loaded as package dependencies. This is why `cryptography` is excluded here for `Emscripten`, and manually loaded in `bumble.js` (the manual loading way supports loading built-in packages)

Reply:
The micropip folks recommended doing this in https://github.com/pyodide/micropip/issues/84. I tested the change with `inv build`, host a static server at the bumble directory, and then navigated to `http://localhost:8000/web/scanner/scanner.html?package=/dist/bumble-0.0.170.dev2%2bg98bf77d.d20230920-py3-none-any.whl` (the last part replaced with the whl file name printed out in `inv build`), and that worked.

(I also locally added `from bumble import crypto` to make sure the import is triggered)

Is there additional testing I should do to verify against the issues you ran into earlier?

Reply:
Ah, good to know! 
The next question we need to answer is: how will pyodide manage its package versions. For now, pyodide comes with `cryptography == 39.x`, but that will evolve. What would be a good strategy to manage compatibility for static pages that load pyodide and want to use the latest Bumble release? Would we need to always pin the version of the Bumble package that the page wants to load, to ensure that the version of pyodide it loads comes with a compatible `cryptography` version?

Reply:
Maybe we should allow a wider range of `cryptography` versions, to allow loading older, compatible versions, yet allow newer versions to be used (from PyPI), even if not yet present in Pyiodide's builtin package list?

Reply:
Good questions. For the first question, I think pinning both the bumble version and the pyodide version would be a good recommendation to ensure compatibility. I wish there is a way to have more relaxed version specification during development that gets frozen when the static HTML is built, but that build step doesn't always exist in general.

For the second question, maybe it makes sense for bumble to be optimistic and make the version `>= 39.0; platform_system=='Emscripten'`, that way it would attempt to use any version of `cryptography` that Pyodide happens to provide us with, the argument being that if there are API breaking changes, it's not worse than failing with "cannot find compatible version of `cryptography`". Having `=='Emscripten'` listed separately also allows other platforms to update its dependency version independently of when Pyodide updates. How does that sound?

Reply:
That sounds good. `cryptography` has been very good with API stability, so I'm not worried about having a loose version match there.

Reply:
Done in the new commit